pages_deployment.go

```diff
@@ -0,0 +1,202 @@
+package cloudflare
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"net/http"
+	"net/url"
+	"strconv"
+	"time"
+
+	"github.com/pkg/errors"
+)
+
+// SizeOptions can be passed to a list request to configure size and cursor location
+// These values will be defaulted if omitted
+type SizeOptions struct {
```

Comment:
we already have a struct for `Cursors` which could be reused here 

https://github.com/cloudflare/cloudflare-go/blob/a6793ca87754dc58afb86cd7a65dd1216b3e786a/cloudflare.go#L361:6

we can add `size` to it if we need it here.

Reply:
(also looks like a typo with `Before` and the JSON tagging)

Reply:
The reason I'm not using this is because it expects strings whereas our endpoint is only ints.

Reply:
The reason I'm not using this is because it expects strings whereas our endpoint is only ints.